### PR TITLE
Don't run GHA on private repo until it is for a PR

### PR DESCRIPTION
**Pull request by maksymmalyhin**

### CODE REVIEW

File: .github/workflows/abtesting.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ on:
 
 jobs:
   pod-lib-lint:
+    # Don't run on private repo unless it is a PR.
```
Comment (paulb777) [inline .github/workflows/abtesting.yml:20]: Also need to delete the on: push: above - here and other .ymls that have it.

Comment (maksymmalyhin) [reply .github/workflows/abtesting.yml:20]: Do you mean we would like to get rid of post-submit checks even in public repo?

Comment (paulb777) [reply .github/workflows/abtesting.yml:20]: Yes, they're inconsistently used and not a lot of value.

Comment (maksymmalyhin) [reply .github/workflows/abtesting.yml:17]: done

**Comment by maksymmalyhin** [comment]
> Did you cancel all the GHA runs for the last commit? If so, did you find a way other than individually cancelling them in the console?

I did and it took some time actually. I found using [this view](https://github.com/firebase/firebase-ios-sdk/actions) the fastest way, but it is far from ideal.

**Pull request merged**